src/app/legacy/containers/CpsRecommendations/index.jsx

```diff
@@ -72,6 +65,10 @@ const CpsRecommendations = ({ items }) => {
 
   if (!hasStoryRecommendations || !enabled || !items.length) return null;
 
+  const titlePath = ['Recommended stories', ['recommendationTitle']];
+
+  const title = pathOr(...titlePath, translations);
```

Comment:
While we're in here, can we try and make this read a bit better (one step closer to getting `rambda` in the 🗑️ 

 ```suggestion
  const title = translations?.recommendationTitle ?? 'Recommended stories';
```

Reply:
pathOr has been 🗑️'ed 😅